### ISSUE TITLE
Document installation steps

### ISSUE THREAD

**Issue by loj** [psychoinformatics-de/datalad-debian#70]
Also include a list of dependencies:
  - singularity 3
  - dget

**Comment by mslw** [comment]
For including Singularity 3 in the list of dependencies, this replaces #67

**Comment by mslw** [comment]
And for dget - I understand that on Debian it's not standalone and prpbably needs to be installed as part of `devscripts`

**Comment by aqw** [comment]
Correct, `dget` is only available as part of the `devscripts` package.
